core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -410,7 +410,8 @@ public Builder setDefaultSupplier(Supplier<Object> supplier, String description)
      * @return the updated builder instance
      */
     public Builder setDefaultValue(Object defaultValue) {
-      checkArgument(validateValue(defaultValue, mType, mEnumType, mValueValidationFunction));
+      checkArgument(validateValue(defaultValue, mType, mEnumType, mValueValidationFunction),
+          "default value " + defaultValue + " of " + mName + " validate failed");
```

Comment:
nit: Is it possible to make it a parameterized string instead of a concatenated string?
https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html

Reply:
Is it better if I change it to `String.format("default value  %s of %s validate failed", defaultValue, mName)`?